src/axom/quest/readers/ProEReader.cpp

```diff
@@ -149,7 +148,7 @@ void ProEReader::setInclusiveBoundingBox(BBox3D& box)
 }
 
 //------------------------------------------------------------------------------
-void ProEReader::compact_arrays(std::vector<bool>& retain_vertex, int elt_count)
+void ProEReader::compact_arrays(slam::BitSet& retain_vertex, int elt_count)
```

Comment:
As far as I can tell, `retain_vertex` is not currently being used in this function or elsewhere.

Was there an intention to reorganize the vertex/tet arrays in this function to remove unused verts?
If so, is there a plan to use it in the future?
If not, does Pro/E reader need the bitset?

Please add a note about this or remove the node bitset from this class if it will not be used.

Reply:
When a ProEReader reads in a file, it reads in all the nodes, then reads and tests each tet.  If the tet doesn't pass the predicate, the algorithm discards the tet.  If the tet passes the predicate, it gets added to the list of tets and the four nodes in the tet are flagged as "used."  So currently, the mesh produced  by the ProEReader contains all tets that pass the predicate and all nodes specified in the file, even the ones not used by any tet.

The algorithm passes the array of flags to `compact_arrays()` because if we want to remove unused nodes, we should do it in `compact_arrays()`.  

What do reviewers think?  Should we squeeze out the unused nodes and renumber all the tets?  In that case, I'll add the functionality here.  If we don't care, I'll remove `retain_vertex` and simplify things.

Reply:
On further discussion, we'll remove the BitSet, since it's diminishing returns for a lot more care and tricky bookkeeping to renumber the vertices.

Reply:
Removed.